message_parser/src/main/java/cumulus_message_adapter/message_parser/MessageParser.java

```diff
@@ -76,9 +79,18 @@ private String HandleMessage(String input, Context context, ITask task, SchemaLo
         }
         catch(Exception e)
         {
+            String errorOutput = e.getMessage();
+            // If necessary, capture stack trace as error output for logging.
+            if (errorOutput == null) {
+                StringWriter sw = new StringWriter();
+                PrintWriter pw = new PrintWriter(sw);
+                e.printStackTrace(pw);
+                errorOutput = sw.toString();
+            }
+
```

Comment:
Don't you want to use `errorOutput` on the line below, instead of `e.getMessage()`?

Reply:
The "line below" isn't visible in the diff because you didn't change it (but should).